JSDemos/Demos/TreeList/LoadDataOnDemand/description.md

```diff
@@ -1 +1,9 @@
-This example demonstrates how to&nbsp;load data on&nbsp;demand every time a&nbsp;user expands a&nbsp;TreeList node. For this purpose, filtering is&nbsp;delegated to&nbsp;the server, and IDs of&nbsp;expanded nodes are sent to&nbsp;the server with every request for data.
\ No newline at end of file
+If the TreeList uses a remote dataset, it can be loaded dynamically as a user expands nodes. Applies only if data has a [plain structure](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataStructure). To configure this feature on the client, do the following:
```

Comment:
- "If the TreeList uses a remote dataset, it can be loaded dynamically" - ambiguous "it". I replaced the entire "if" phrase with a statement.
- "Applies only if data has a [plain structure](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataStructure)." - such incomplete sentences are great for short descriptions in the API reference, but you shouldn't use them in long texts.
- "To configure this feature on the client" - we haven't mentioned that there will be a client- and a server-side configuration.

```suggestion
The TreeList can load a remote dataset dynamically as a user expands nodes. The dataset should have a [plain structure](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataStructure).

This feature requires client- and server-side configuration. To configure the client-side part, do the following:
```